### PR TITLE
Installation section improvements

### PR DESCRIPTION
**Pull request by lfarah**
Got it from my [iOS Readme Template](https://github.com/awesome-labs/iOS-readme-template)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,13 +20,27 @@ Prerequisite
...
-* Manual
-  - Copy `CHTReachability.h/m` to your project.
-  - Copy `Vender` folder to your project.
-  - Add `SystemConfiguration.framework`.
+#### CocoaPods
+You can use [CocoaPods](http://cocoapods.org/) to install `CHTReachability` by adding it to your `Podfile`:
+
+```ruby
+platform :ios, '8.0'
+use_frameworks!
```
Comment (chiahsien) [inline README.md:29]: I think there should be a comment indicating this line is for Swift.

File: README.md [hunk 2]
```diff
@@ -20,13 +20,27 @@ Prerequisite
...
+pod 'CHTReachability'
+```
+
+To get the full benefits import `CHTReachability` wherever you import UIKit
+
+``` swift
+import UIKit
+import CHTReachability
+```
```
Comment (chiahsien) [inline README.md:39]: Should be an `objc` version, too.

**Comment by lfarah** [comment]
Can you give me the line for objective-C? It's been a while that I don't do Obj-c...

**Comment by chiahsien** [comment]
@lfarah 

``` objc
#import <CHTReachability/CHTReachability.h>
```

**Comment by lfarah** [comment]
🎉

**Pull request merged**